docs/serving/app-security/security-guard-install.md

```diff
@@ -61,15 +61,153 @@ To start this tutorial, after installing Knative Serving, run the following proc
+    CERT=`kubectl create cm config-deployment --from-file $ROOTCA -o json --dry-run=client |jq .data.\"$FILENAME\"`
+
+    echo "Get the current config-deployment configmap"
+    CURRENT="$(mktemp)"
+    kubectl get cm config-deployment -n knative-serving -o json | jq 'del(.data, .binaryData | ."queue-sidecar-token-audiences", ."queue-sidecar-rootca" )' > $CURRENT
+
+    echo "Add queue-sidecar-token-audiences"
+    AUDIENCES="$(mktemp)"
+    jq '.data |= . + { "queue-sidecar-token-audiences": "guard-service"}' $CURRENT > $AUDIENCES
+
+    echo "Join the two config-deployment configmaps into one"
+    MERGED="$(mktemp)"
+    jq  --arg cert "${CERT}" '.data  |= . + { "queue-sidecar-rootca": $cert}' $AUDIENCES > $MERGED
+
+    echo "Apply the joined config-deployment configmap"
+    kubectl apply -f $MERGED -n knative-serving
+
+    echo "cleanup"
+    rm $MERGED $AUDIENCES $ROOTCA $CURRENT
+
+    echo "Results:"
+    kubectl get cm config-deployment -n knative-serving -o json|jq '.data'
+    ```
+
+    Use the following script to unset TLS in `config-deployment`:
+
+    ```
+    echo "Get the current config-deployment configmap"
+    CURRENT="$(mktemp)"
+    kubectl get cm config-deployment -n knative-serving -o json | jq 'del(.data, .binaryData | ."queue-sidecar-token-audiences", ."queue-sidecar-rootca" )' > $CURRENT
+
+    echo "Apply the joined config-deployment configmap"
+    kubectl apply -f $CURRENT -n knative-serving
+
+    echo "cleanup"
+    rm $CURRENT
+
+    echo "Results:"
+    kubectl get cm config-deployment -n knative-serving -o json|jq '.data'
+    ```
+
+=== "Using Knative Operator"
+
+    Use the following alternative script to install Security-Guard with TLS and Serving using the Knative Operator:
+
+    ```
+    echo "Copy the certificate to file"
+    ROOTCA="$(mktemp)"
+    FILENAME=`basename $ROOTCA`
+    kubectl get secret -n knative-serving knative-serving-certs -o json| jq -r '.data."ca-cert.pem"' | base64 -d >  $ROOTCA
+
+    echo "Create a temporary config-deployment configmap with the certificate"
+    CERT=`kubectl create cm config-deployment --from-file $ROOTCA -o json --dry-run=client |jq .data.\"$FILENAME\"`
+
+    echo "cleanup"
+    rm $ROOTCA
+
+    kubectl apply --filename - <<EOF
+    apiVersion: v1
+    kind: Namespace
+    metadata:
+    name: knative-serving
+    ---
+    apiVersion: operator.knative.dev/v1beta1
+    kind: KnativeServing
+    metadata:
+    name: knative-serving
+    namespace: knative-serving
+    spec:
+    deployments:
+    - name: guard-service
+        env:
+        - container: guard-service
+        envVars:
+        - name: GUARD_SERVICE_TLS
+            value: "true"
+        - name: GUARD_SERVICE_AUTH
+            value: "true"
+    security:
+        securityGuard:
+        enabled: true
+    ingress:
+        kourier:
+        enabled: true
+    config:
+        network:
+        ingress.class: "kourier.ingress.networking.knative.dev"
```

Comment:
Line 203 to Line 208 are configuration for ingress, do we need to add them for the security guard?